src/SelfRoleManagerEvents.ts

```diff
@@ -14,6 +14,7 @@ export enum SelfRoleManagerEvents {
   interaction = 'interaction',
 
   maxRolesReach = 'maxRolesReach',
+  conditionsNotMet = 'conditionsNotMet',
```

Comment:
Am I indecisive ? I don't know... But I would be more open onto what this event is really _meaning_. Should thus probably be renamed to something like **`requiredRolesMissing`** instead of `conditionsNotMet`. Sorry for the double change!! 😭

Reply:
Don't worry, it's a better name!